packages/core/src/components/forms/_label.scss

```diff
@@ -111,14 +135,24 @@ label.pt-label {
   }
 
   &.pt-disabled {
-    color: $pt-text-color-disabled;
+    &,
+    .pt-label-description {
+      color: $pt-text-color-disabled;
+    }
   }
 
   .pt-dark & {
     color: $pt-dark-heading-color;
 
+    .pt-label-description {
+      color: $pt-text-color-muted;
+    }
+
     &.pt-disabled {
-      color: $pt-dark-text-color-disabled;
+      &,
+      .pt-label-description {
```

Comment:
`.pt-text-muted { color: $pt-text-color-disabled; }` isn't that nice either. Is there an elegant way to get rid of `.pt-label-description`?